src/ralph/models/edx/converters/xapi/server.py

```diff
@@ -0,0 +1,55 @@
+"""Server event xAPI Converter"""
+
+from ralph.models.xapi.constants import (
+    EXTENSION_COURSE_ID,
+    EXTENSION_MODULE_ID,
+    EXTENSION_SCHOOL_ID,
+)
+from ralph.models.xapi.page_viewed import PageViewed
+
+from .base import BaseXapiConverter
+
+
+class ServerEventToXapi(BaseXapiConverter):
+    """Converts a common edX server event to xAPI.
+
+    Example Statement: John viewed https://www.fun-mooc.fr/ Web page.
+
+    WARNING: The converter does not include the following edX fields as they may
+    contain sensitive data: `username`, `referer`, `event`, `event_source`, `ip`,
+    `agent`, `accept_language:`, `context.course_user_tags`
+
+    WARNING: The converter may not include the following edX fields:
+        - context.org_id: When `org_id` is an empty string.
+        - context.course_id: When `course_id` is an empty string.
+    """
+
+    __model__ = PageViewed
```

Comment:
This might be confusing, indeed. The model of the converter is the destination model. Maybe it's worth renaming it to `__destination__` or `__result__` ?

Reply:
What is confusing is that your converter is called `ServerEventToXapi` for a destination model that is a `PageViewed` event.

Reply:
Good point) Thanks) I agree) will rename the class to `ServerEventToPageViewed`)